.github/workflows/snyk-lts.yml

```diff
@@ -0,0 +1,50 @@
+# This workflow will scan and monitor all the LTS releases for vulnerabilities with Snyk 
+name: Snyk Container for LTS Releases
+on:
+  push:
+    # These branches represent the LTS releases
+    branches:
+      - 0.12.[0-9]+
+      - 0.11.[0-9]+
```

Comment:
Is the idea to keep an active branch for each LTS `minor` version? Could we have a `x.x-lts` branch instead that starts at the time a release is designated as LTS and continues until support is dropped?

Reply:
Yes, but we still need to have an explicit identifier on each patch.

Reply:
We will have a tag for each release we create, so that we can go back to that point at any time. I think that having multiple branches for each minor could cause us to get alerts for old branches for which a newer release has already fixed the vulnerabilities.

Reply:
This workflow will kick off only if there is a push to the branch. Example 0.12.2 is released, it will be added to Snyk monitor i.e. new project is created in Synk account. 
Once the newer patch version of LTS (ex. 0.12.2) is added to the monitor, older patch version (0.12.1) needs to be removed from monitoring by deleting the project in Snyk account.

Reply:
I feel like this might generate operational overhead since we have to track/manage branches in two places (the repo and the Snyk project) - hence the idea of having a `-lts` branch for the `major.minor` version designated as such. It would keep being updated for the life of the LTS release, tags will be created for each minor release to have point-in-time traceability and it will only require maintainers to go and delete the project in Snyk (unsure on how to access that at this time) once the LTS status expires.

Reply:
There will be exactly one long-lasting branch per LTS minor release, and all of the patch releases will be from that branch.  We currently have `0.11.x` and `0.12.x` branches for the 0.11 and 0.12 LTS, respectively — although we are planning to rename them to `0.11-lts` and `0.12-lts`.  They will be created at the latest patch created at the time we declare it an LTS.  So, for example:

- Release a minor version — e.g. 1.0.0
- Wait a bit for it to stabilize — perhaps a 1.0.1, and 1.0.2, or even a 1.1.0
- Declare it LTS and create the “1.1.lts” branch at tag 1.1.0
- 1.1.1 and so on are all on the LTS branch.

As such, the synk yml only needs to be updated when we declare an LTS, and removed when we end of life it.